pkg/scheduler/objects/queue.go

```diff
@@ -129,8 +130,7 @@ func NewDynamicQueue(name string, leaf bool, parent *Queue) (*Queue, error) {
 	}
 	// name might not be checked do it here
 	if !configs.QueueNameRegExp.MatchString(name) {
-		return nil, fmt.Errorf("invalid queue name %s, a name must only have alphanumeric characters,"+
-			" - or _, and be no longer than 64 characters", name)
+		return nil, fmt.Errorf("invalid queue name '%s', a name must only have alphanumeric characters, - or _, and be no longer than 64 characters", name)
```

Comment:
Is this line too long?

Reply:
Unrelated to this PR, but how come we set the limit to 64? In Kubernetes, 63 is the limit (e.g. for labels)

Reply:
It is not a label value or name it is a queue name. K8s labels are different in a number of ways.  The major difference is that label values and names can contain dots, queue names cannot contain dots. It looks similar but it is not and we do not want to be either.